xml/System.Text/Rune.xml

```diff
@@ -668,10 +678,19 @@ This method is very similar to <xref:System.Text.Rune.DecodeFromUtf8(System.Read
         <Parameter Name="value" Type="System.Text.Rune" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The <see cref="T:System.Text.Rune" /> to evaluate.</param>
+        <summary>Indicates whether the specified <see cref="T:System.Text.Rune" /> is categorized as a letter.</summary>
+        <returns><see langword="true" /> if <paramref name="value" /> is a letter; otherwise <see langword="false" />.</returns>
+        <remarks>
```

Comment:
Can you please wrap the remark comments like in my suggestions above? It might be easier directly in code than with suggestions:

```xml
        <remarks>
          <format type="text/markdown"><![CDATA[

## Remarks

// comments aligned to the left. Instead of <see cref="API" />, use <xref:API>

          ]]></format>
        </remarks>
```